wavefront-hpa-adapter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.9.12
```

Comment:
i guess since there are no image upgrades, we don't need to bump the app version ?

Reply:
revert appVersion to 0.9.12